CONTRIBUTING.md

```diff
@@ -18,6 +18,7 @@ You can do this online, and it only takes a minute.
 
 ## Workflow for making changes
 
+- Switch to flutter master channel `flutter channel master`
```

Comment:
```suggestion
- Change flutter to the latest flutter candidate:
    `./tool/update_flutter_sdk.sh --local`
```

Reply:
This helper was made to switch to the version specified by ./tool/latest_flutter_candidate.sh, which is the version that we build Devtools against :D